charts/gitops-server/templates/role.yaml

```diff
@@ -22,6 +22,9 @@ rules:
     {{- with .Values.rbac.viewSecrets }}
     resourceNames: {{ . | toJson }}
     {{- end }}
+    {{- with .Values.rbac.viewSecretsResourceNames }}
```

Comment:
This should error if both `viewSecrets` and `viewSecretsResourceNames` are set (or we breaking-bump the version...)